mne/annotations.py

```diff
@@ -230,6 +230,13 @@ class Annotations(object):
              n                        |      |
              e                        +------+
          orig_time                 onset[0]'
+
+    **Specific annotation**
```

Comment:
```suggestion
    .. note::
```

Reply:
then you will need to check that it renders well. Otherwise a section `Notes`

Reply:
It renders ok. I did that in the first place because the ``Notes`` section of ``mne.Annotations`` is quit large and has already 2 paragraphs (+ the intro) named **ch_names** and **orig_time**. So adding **Specific annotation** would keep the format, but if you think adding a note is better, I'm ok with it. Your call :)

Reply:
it appears in bold in the html sphinx rendered doc?

Reply:
Yes, as sub-section titles of the section ``Notes``. Here: https://mne.tools/dev/generated/mne.Annotations.html